src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -30,4 +30,75 @@ public ParseFormulasResult(IEnumerable<KeyValuePair<IdentToken, TexlNode>> named
             NamedFormulas = namedFormulas;
         }
     }
+
+    internal class ParseUDFsResult
+    {
+        internal IEnumerable<UDF> UDFs { get; }
+
+        internal List<TexlError> Errors { get; }
```

Comment:
Why is the original TexlError list needed? Why not convert to expression errors immediately and store them here instead?

Reply:
Originally did it to emulate how ParsedFormulas was structured, now its removed and just has the expression errors.